sessions/js-createelement/product-teaser/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+		<title>HTML & CSS & JS</title>
+		<link rel="stylesheet" href="./css/styles.css" />
+		<script src="./js/index.js" defer></script>
+	</head>
+	<body>
+		<h1>Products</h1>
+		<article class="product">
+			<div class="product__body">
+				<div class="product__text-container">
+					<h2 class="product__name">Anemonefish</h2>
+					<ul class="product__categories">
+						<li class="product__category">Sea water</li>
+						<li class="product__category">Warmer waters</li>
+						<li class="product__category">Omnivorous</li>
+					</ul>
+					<p class="product__description">
+						Anemonefish are omnivorous and can feed on undigested food from their host
+						anemones. When a sea anemone is not available in an aquarium, the
+						anemonefish may settle in some varieties of soft corals, or large polyp
+						stony corals.
+					</p>
+				</div>
+				<div class="product__image-container">
+					<img
+						class="product__image"
+						src="https://unsplash.com/photos/rEM3cK8F1pk/download?ixid=MnwxMjA3fDB8MXxzZWFyY2h8NHx8bmVtb3xlbnwwfHx8fDE2NTk1MjE1MzA&force=true&w=640"
+						alt="Side-view of an anemonefish"
```

Comment:
I would probably leave the alt attribute empty as it is merely a descriptive image and does not show crucial information to understand the content of the website.
```suggestion
						alt=""
```

Reply:
Good Idea!